src/lib/extended/style/style.ts

```diff
@@ -55,12 +55,19 @@ export class StyleDirective extends BaseDirective2 implements DoCheck {
     this.fallbackStyles = this.buildStyleMap(styles);
   }
 
+  /** Add generated styles */
   protected updateWithValue(value: any) {
     const styles = this.buildStyleMap(value);
     this.ngStyleInstance.ngStyle = {...this.fallbackStyles, ...styles};
     this.ngStyleInstance.ngDoCheck();
   }
 
+  /** Remove generated styles */
+  protected clearStyles() {
+    this.ngStyleInstance.ngStyle = this.fallbackStyles;
```

Comment:
I think this will fix a bug we saw with ngStyle not updating on window resize that I hadn't gotten a chance to create a report for :)